skuba-update/skuba_update/skuba_update.py

```diff
@@ -289,19 +290,22 @@ def run_zypper_command(command, needsOutput=False):
     Run the given zypper command. The command is expected to be a tuple which
     also contains the 'zypper' string. It returns the exit code from zypper.
     """
+    # updateProcessId = 'skuba-update-{0}'.format(os.getpid())
+    uniqueId = 'skuba-update-{0}'.format(uuid.uuid4())
+    if command[0] != 'zypper':
```

Comment:
Will drop this if statement before final commit.

Reply:
Why random rather than a hostname+timestamp `uuid1()`?  Any particular reason?